site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -289,4 +342,62 @@ public function deleteQueue($queue_code) {
             new RedirectResponse($this->core->buildCourseUrl(['office_hours_queue']))
         );
     }
+
+    /**
+    * @Route("/{_semester}/{_course}/office_hours_queue/checkUpdates", methods={"GET"})
```

Comment:
`checkUpdates` is inconsistent with `change_token` and `current_queue` below.

```suggestion
    * @Route("/{_semester}/{_course}/office_hours_queue/check_updates", methods={"GET"})
```